config_converter/config_mapper/config_mapper.py

```diff
@@ -48,21 +84,34 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> dict:
             sys.exit()
         plugin_name = plugin_prefix_map[d.name] + plugin_type
         if plugin_name not in _SUPPORTED_PLUGINS:
+            stats['entities_unrecognized'] += 1
+            stats['attributes_unrecognized'] += get_num_attrs(d)
             print(f'We do not know plugin {plugin_name}')
         else:
             plugin_dir = dir_name_map[d.name]
             if plugin_dir not in logs_module:
                 logs_module[plugin_dir] = []
-            logs_module[plugin_dir].append(_convert_plugin(d, plugin_name))
+            old_attrs = stats['attributes_recognized']
+            logs_module[plugin_dir].append(
+                _convert_plugin(d, plugin_name, stats))
+            current_dir_attrs: int = get_num_attrs(d)
+            if old_attrs + current_dir_attrs == stats['attributes_recognized']:
+                stats['entities_recognized_success'] += 1
+            elif old_attrs == stats['attributes_recognized']:
+                stats['entities_recognized_failure'] += 1
+            else:
+                stats['entities_recognized_partial'] += 1
             try:
                 result['logging_level'] = next(p.value for p in d.params
                                                if p.name == '@log_level')
+                stats['attributes_recognized'] += 1
             except StopIteration:
                 continue
+    print(json.dumps(stats))
```

Comment:
So would you suggest printing in the main function instead? I was initially thinking of putting in tests in the next PR - but I can do it in this one instead since I think that would make more sense. Thanks!